client/src/utils/prettifyStatus.js

```diff
@@ -25,7 +25,7 @@ export const getParticipantStatusData = (status, tabValue, isMoH, participantInf
   const isPendingAcknowledgement = status.includes('pending_acknowledgement');
   const isWithdrawn = status.includes('withdrawn');
   const isArchived = status.includes('archived');
-  const rosStartDate = isROS ? participantInfo.rosStatuses[0].data.date : null;
+  const rosStartDate = isROS ? participantInfo?.rosStatuses[0]?.data?.date : null;
```

Comment:
This is unrelated to HCAP-1308, I just thought it was a minor change that I could piggyback here. Just some null checks. I couldn't reproduce Akhil's crashing issue, but this seems like a good idea regardless.